pkg/cosign/kubernetes/webhook/clusterimagepolicy/clusterimagepolicy_types.go

```diff
@@ -15,12 +15,25 @@
 package clusterimagepolicy
 
 import (
+	"context"
+	"crypto"
 	"crypto/ecdsa"
 	"crypto/x509"
 	"encoding/json"
 	"encoding/pem"
+	"fmt"
+	"strings"
 
 	"github.com/sigstore/cosign/pkg/apis/cosigned/v1alpha1"
+	"github.com/sigstore/cosign/pkg/apis/utils"
```

Comment:
nit: separate the groups of imports as groups (sigstore ones, external, golang packages).